packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -121,8 +123,10 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage> {
             mainAxisSize: MainAxisSize.max,
             children: <Widget>[
               Flexible(
-                child: Text(widget.title, overflow: TextOverflow.fade),
-              ),
+                  child: Text(
+                widget.title,
+                overflow: TextOverflow.fade,
+              )),
```

Comment:
Another `,` between those two would be good

Reply:
hi @M123-dev  I have added , .